DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_RadioQuestion.cshtml

```diff
@@ -2,46 +2,50 @@
 @model AssessmentQuestion
 
 <div class="nhsuk-u-margin-bottom-4 nhsuk-u-margin-top-6">
-  <fieldset class="nhsuk-fieldset">
-    <legend class="nhsuk-fieldset__legend nhsuk-fieldset__legend--m">
-      <h3 class="nhsuk-fieldset__heading">
-        @Model.Question @if (!Model.Required)
-        {<span>(optional)</span>}
-      </h3>
-      <div class="nhsuk-hint">
-        @if (Model.ScoringInstructions != null)
-        {
-          <p>@(Html.Raw(Model.ScoringInstructions))</p>
-        }
-        <p>Please pick one answer.</p>
-      </div>
-    </legend>
+    <fieldset class="nhsuk-fieldset">
+        <legend class="nhsuk-fieldset__legend nhsuk-fieldset__legend--m nhsuk-u-margin-bottom-0">
+            <h3 class="nhsuk-fieldset__heading">
+                @Model.Question @if (!Model.Required)
+                {
+                    <span>(optional)</span>
+                }
+            </h3>
+            <div class="nhsuk-hint nhsuk-u-margin-top-3">

```

Comment:
Have you checked that this doesn't introduce additional spacing when `Model.ScoringInstructions != null` ?

Reply:
yes I've checked it now and it does not introduce additional spacing when scoring instructions is not null. Screenots are attached in the jira